server/src/startRating.js

```diff
@@ -0,0 +1,8 @@
+// console.log("heyo");
+// updateOrCreateReview(update: UpdateReview!, create: CreateReview!): Review
+export default event => {
+  console.log(event.data);
```

Comment:
remove this console log.

Reply:
These entire files were supposed to be deleted. I don't know how they came back to life. They've been deleted again.